### ISSUE TITLE
Release v2.1?

### ISSUE THREAD

**Issue by karllark** [PAHFIT/pahfit#172]
The milestone for v2.1 shows no open issues or pull requests as of a few minutes ago.  Please comment on this issue if you agree v2.1 is ready for release.  Or what more is needed if not.

https://github.com/PAHFIT/pahfit/milestone/3

https://github.com/PAHFIT/pahfit/wiki/Planning#version-21

**Comment by jdtsmith** [comment]
Yes.

**Comment by els1** [comment]
yes

**Comment by jdtsmith** [comment]
But we should probably fix #174 first.

**Comment by drvdputt** [comment]
Yes, this should probably be a new version. Not only because of the API changes, but also because the instrument packs are an entirely new concept, changing something fundamental about the fitting.
